src/app/replicator/change/change_contact_handler.coffee

```diff
@@ -79,3 +80,20 @@ module.exports = class ChangeContactHandler
         , cb
         , new ContactFindOptions cozyId, false, [], AndroidAccount.TYPE, \
             AndroidAccount.NAME
+
+
+    _setPictureBase64data: (doc, callback) ->
+        unless doc._attachments or 'picture' of doc._attachments
+            return callback doc
+        return callback doc if typeof doc._attachments.picture.data is 'string'
+
+        reader = new FileReader()
+        reader.onload = =>
+            data = reader.result
+            if data.startsWith 'data:application/octet-stream;base64,'
+                data = data.substr 37
```

Comment:
This 37 is a magic number. You should probably do something like:

```coffee
prefix = 'data:application/octet-stream;base64,'
if data.startsWith prefix
    data = data.substr prefix
```

It's better for future work. For example, if you discover that a new version of pouchdb does not put the space at the end of the string, you will have only to modify the string, not to recompute yourself the length.